### PR TITLE
Add Advantech interface epc_t4286-1.json

### PR DESCRIPTION
**Pull request by paul53huang**
Add Advantech interface epc_t4286-1.json for Pnp certification.

### CODE REVIEW

File: dtmi/advantech/epc_t4286-1.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      "name": "VGA",
+      "schema": "dtmi:Advantech:VGA;1"
+    },
+    {
+      "@type": "Component",
+      "name": "SDRAM",
+      "schema": "dtmi:Advantech:SDRAM;1"
+    }
+  ]
+}
```
Comment (advfredchang) [inline dtmi/advantech/epc_t4286-1.json:38]: All requirements met.

**Review by advfredchang — APPROVED** [review]

**Comment by paul53huang** [comment]
Remove this pull-request.

**Comment by paul53huang** [comment]
Reopen.

**Review by rido-min — APPROVED** [review]
LGTM

**Pull request merged**